VAMobile/e2e/tests/Appeals.e2e.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, device, by, element} from 'detox'
+import {loginToDemoMode, openBenefits, openClaims, openClaimsHistory } from './utils'
+import { setTimeout } from 'timers/promises'
+import { DateTime } from 'luxon'
+
+export const AppealsIdConstants = {
+  APPEAL_1_ID: 'Disability compensation appeal updated on November 22, 2011 Submitted June 12, 2008',
+  REVIEW_PAST_EVENTS_ID: 'reviewPastEventsTestID',
+  ISSUES_TAB_TEXT: 'Issues',
+  STATUS_TAB_TEXT: 'Status',
+  APPEALS_DETAILS_ID: 'appealsDetailsTestID',
+  APPEAL_DETAILS_TEXT: 'Appeal details',
+  APPEAL_TYPE_TEXT: 'Appeal for compensation',
+  APPEAL_SUBMITTED_TEXT: 'Submitted June 12, 2008',
+  APPEAL_NEED_HELP_NUMBER_TEXT: '800-827-1000',
+  APPEAL_VISIT_VA_TEXT: 'Visit VA.gov',
+}
+
+export async function getDateWithTimeZone(dateString: string) {
```

Comment:
Consider making this a global utility function since it's already in another test file as well (and I assume more in the future)